templates/tomcat/login.config

```diff
@@ -0,0 +1,13 @@
+CertificateLogin {
```

Comment:
If there are no variables (as it is now), it can also live in `files/` and you can use `file()`. Otherwise I prefer the `.erb` filename suffix to distinguish them from `epp` templates.